Makefile

```diff
@@ -178,4 +178,7 @@ build_python_host: deps_python_host ${CORE_WASM}
 test_python_host: build_python_host ${TEST_CORE_WASM}
 	cp ${TEST_CORE_WASM} ${PYTHON_HOST_ASSETS}/test-core.wasm
 	cd packages/python_host; source venv/bin/activate; \
-	python3 -m unittest discover tests/
\ No newline at end of file
+	python3 -m unittest discover tests/
+
+build_map_std_package: ${MAP_STD}
+	cp -r core_js/map-std/types packages/map_std/
```

Comment:
I'll look into it. It should work.

Reply:
Turns out it doesn't work. Symlink for common_js works because we run `tsc` on it and the transpiled files are generated from it.

Here we want to directly package the generated `types` folder, so we need to copy it, otherwise we would only package a symlink.

Reply:
🤦 yep, that makes sense. Sorry for missleading you.